dragonflybsd/DragonFlyBSDProcessList.c

```diff
@@ -261,29 +261,68 @@ static inline void DragonFlyBSDProcessList_scanMemoryInfo(ProcessList* pl) {
    pl->usedSwap *= pageSizeKb;
 }
 
-static char* DragonFlyBSDProcessList_readProcessName(kvm_t* kd, const struct kinfo_proc* kproc, int* basenameEnd) {
+//static void DragonFlyBSDProcessList_updateExe(const struct kinfo_proc* kproc, Process* proc) {
+//   const int mib[] = { CTL_KERN, KERN_PROC, KERN_PROC_PATHNAME, kproc->kp_pid };
+//   char buffer[2048];
+//   size_t size = sizeof(buffer);
+//   if (sysctl(mib, 4, buffer, &size, NULL, 0) != 0) {
+//      Process_updateExe(proc, NULL);
+//      return;
+//   }
+//
+//   /* Kernel threads return an empty buffer */
+//   if (buffer[0] == '\0') {
+//      Process_updateExe(proc, NULL);
+//      return;
+//   }
+//
+//   Process_updateExe(proc, buffer);
+//}
```

Comment:
Why not use this as fall-back when reading procfs fails with a note mentioning [this patch for the kernel bug](https://gitweb.dragonflybsd.org/dragonfly.git/commitdiff/a23ed7f524eb73f2bf9b81db08b87f033b94e6d3)?

One thing we should avoid is keeping dead code in the repository.